core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncer.java

```diff
@@ -78,6 +80,7 @@ public ActiveSyncer(FileSystemMaster fileSystemMaster, ActiveSyncManager syncMan
   @Override
   public void heartbeat() {
     LOG.debug("start sync heartbeat for {} with mount id {}", mMountUri, mMountId);
+    AlluxioEvent.ActiveSyncStarted.fire();
```

Comment:
I feel like these active sync events will be noisy. Do we need them now?

Reply:
why theyb would be noisy?

Reply:
Isn't it syncing on a heartbeat? We would see this event on every tick, which could be frequent and noisy. I don't think we want an event associated with a heartbeat tick.

Reply:
removed events.